client/thin-replica-client/src/thin_replica_client.cpp

```diff
@@ -957,6 +958,9 @@ void ThinReplicaClient::Subscribe() {
     state.pop_front();
   }
   latest_verified_block_id_ = block_id;
+  metrics_.last_verified_block_id.Get().Set(latest_verified_block_id_);
+  metrics_.current_queue_size.Get().Set(config_->update_queue->size());
```

Comment:
Would operator= from Gauge work here?

Reply:
We overload operator= for another gauge, but here I try to set a value uint64_t and we don't overload it.